lib/bronto/contact.rb

```diff
@@ -20,11 +20,11 @@ def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, includ
       Array.wrap(resp[:return]).map { |hash| new(hash) }
     end
 
-    def self.save(*objs)
+    def self.save(endpoint, *objs)
```

Comment:
Do the tests for this library pass as-is? If so, then let's keep them passing. If they are busted, I'm not sure it's worth the trouble.

Do we use this method in the site's code a lot?